lib/src/konsistTest/kotlin/com/lemonappdev/konsist/api/ApiKonsistTest.kt

```diff
@@ -18,7 +18,19 @@ class ApiKonsistTest {
             .assert { it.hasReturnType() }
     }
 
+    @Test
+    fun `every api declaration has valid KDoc`() {
+        testScope
+            .declarations(includeNested = true, includeLocal = true)
+            .assert { it.hasValidKDoc(verifyParamTag = true, verifyReturnTag = true) }
+    }
+
     companion object {
         val apiPackageScope = Konsist.scopeFromPackage("com.lemonappdev.konsist.api..", sourceSetName = "main")
+
+        // change `testScope` to `apiPackageScope` in above test and remove `testScope` property
+        val testScope =
+            Konsist.scopeFromProjectFile("lib/src/main/kotlin/com/lemonappdev/konsist/api/declaration/KoNamedDeclaration.kt") +
```

Comment:
I think we can reuse `apiPackageScope` scope to check entire project and generate these docs using AI ;-)